### PR TITLE
release: 2024-01-29

### PR DESCRIPTION
**Pull request by cb22**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,70 @@
...
+
+- [#1460](https://github.com/tigerbeetle/tigerbeetle/pull/1460)
+
+  Stop an isolated replica from locking a standby out of a cluster.
+
+### Features
+
+- [#1470](https://github.com/tigerbeetle/tigerbeetle/pull/1470)
+
+  Change `get_account_transfers` to use `timestamp_min` and `timestamp_max` for easier pagination.
```
Comment (batiati) [inline CHANGELOG.md]: Actually, it made pagination more difficult!

```suggestion
  Change `get_account_transfers` to use `timestamp_min` and `timestamp_max` to allow filtering by time ranges.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,70 @@
...
+
+### Internals
+
+- [#1480](https://github.com/tigerbeetle/tigerbeetle/pull/1480)
+
+  Deprecate support and testing for Node 16, which is EOL.
+
+- [#1461](https://github.com/tigerbeetle/tigerbeetle/pull/1461)
+
+  `tigerbeetle client ...` is now `tigerbeetle repl ...`.
```
Comment (matklad) [inline CHANGELOG.md]: Could you move that to features? This is very much user-visible.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,70 @@
...
+- [#1477](https://github.com/tigerbeetle/tigerbeetle/pull/1477),
+  [#1469](https://github.com/tigerbeetle/tigerbeetle/pull/1469),
+  [#1475](https://github.com/tigerbeetle/tigerbeetle/pull/1475),
+  [#1457](https://github.com/tigerbeetle/tigerbeetle/pull/1457),
+  [#1452](https://github.com/tigerbeetle/tigerbeetle/pull/1452).
+
+  Improve VOPR & VSR logging, docs, assertions and tests.
+
+- [#1474](https://github.com/tigerbeetle/tigerbeetle/pull/1474
+)
```
Comment (matklad) [inline CHANGELOG.md]: extra \n here?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,70 @@
...
+
+  Improve integration tests around Node and `pending_transfer_expired` - thanks to our friends at
+  Rafiki for reporting!
```
Comment (matklad) [inline CHANGELOG.md:62]: :heart:

File: CHANGELOG.md (hunk 3 above)
Comment (cb22) [reply CHANGELOG.md]: Snuck in! Sublime collapses markdown links including newlines :exploding_head:.

**Review by matklad — APPROVED** [review]

**Comment by matklad** [comment]
(moved to merge queue to prevent Rafael from edging ahead of us here https://github.com/tigerbeetle/tigerbeetle/pull/1485 :) )

**Pull request merged**